src/main/java/net/pms/newgui/Splash.java

```diff
@@ -63,6 +61,8 @@ public Splash(PmsConfiguration config) {
 		setLayout(null);
 		add(imglabel);
 		imglabel.addMouseListener(this);
+		img = new ImageIcon(getClass().getResource("/resources/images/icon-32.png"));
+		setIconImage(img.getImage());
```

Comment:
@Nadahar ahh I see. I was loking for the icon in the indicative area of the main panel and missed the main window icon which before your change has shown an Java icon. Good job 👍 
I recommend to merge this PR and I will delete the #1324